sdk/src/main/java/com/mapbox/maps/renderer/widget/WidgetPosition.kt

```diff
@@ -0,0 +1,8 @@
+package com.mapbox.maps.renderer.widget
+
+enum class WidgetPosition {
+  TOP_LEFT,
+  TOP_RIGHT,
+  BOTTOM_LEFT,
+  BOTTOM_RIGHT,
```

Comment:
@yunikkk the current road label is in the bottom center. will there be a way to center the widgets inside a visible region?

Reply:
Good point, will add CENTER_ positions

Reply:
Added center alignment options